src/plugins/vis_types/timeseries/server/lib/vis_data/helpers/bucket_transform.js

```diff
@@ -193,7 +193,7 @@ export const bucketTransform = {
     };
     if (bucket.gap_policy) body.serial_diff.gap_policy = bucket.gap_policy;
     if (bucket.lag) {
-      body.serial_diff.lag = /^([\d]+)$/.test(bucket.lag) ? bucket.lag : 0;
+      body.serial_diff.lag = /^([\d]+)$/.test(bucket.lag) ? bucket.lag : 1;
```

Comment:
`lag` is optional for `ES`. 
I think to avoid of setting the default value you can move `/^([\d]+)$/.test(bucket.lag)` to parent `if`

Reply:
done